webapp/components/test-file-results-table.js

```diff
@@ -165,7 +169,16 @@ class TestFileResultsTable extends WPTFlags(Pluralizer(AmendMetadataMixin(WPTCol
         </template>
       </tr>
     </template>
-
+    <template is="dom-if" if="[[shouldShowTotals(totals)]]">
+      <tr class="totals-row">
+        <td class="sub-test-name"><code>Total</code></td>
+        <template is="dom-repeat" items="[[totals]]" as="totals">
```

Comment:
`<template is="dom-repeat" items="[[totals]]" as="totals">` should be `as=total`. Otherwise you are reassigning the existing `totals` variables here.

Reply:
Or "columnTotal" as `total.toal` is confuisng

Reply:
That's great insight - I did not realize this was happening! I've change it as you suggested